build.gradle

```diff
@@ -64,10 +63,10 @@ dependencies {
 	}
 	implementation 'net.sf.jopt-simple:jopt-simple:5.0.3'
 
-	testCompileOnly 'org.jetbrains:annotations:19.0.0'
+	testImplementation 'org.jetbrains:annotations:20.1.0'
```

Comment:
This should get added via loom, can prob just remove it

Reply:
resolved in https://github.com/FabricMC/fabric-loader/pull/419/commits/bfb0feea74355d1b2f38eb39f131bcc3727c51b1

Reply:
Hmm, workflow fails on my fork when the annotations are removed

Reply:
I think you need to bump to like loom 0.8.7